webpack/scenes/RedHatRepositories/components/RecommendedRepositorySetsToggler.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Switch, Icon, FieldLevelHelp } from 'patternfly-react';
+
+import './RecommendedRepositorySetsToggler.scss';
+
+const RecommendedRepositorySetsToggler = ({
+  enabled,
+  className,
+  children,
+  help,
+  onChange,
+  ...props
+}) => {
+  const classes = classNames('recommended-repositories-toggler-container', className);
+
+  return (
+    <div className={classes} {...props}>
+      <Switch bsSize="mini" value={enabled} onChange={() => onChange(!enabled)} />
+      <Icon type="fa" name="star" />
+      {children}
+      <FieldLevelHelp content={help} />
+    </div>
+  );
+};
+
+RecommendedRepositorySetsToggler.propTypes = {
+  enabled: PropTypes.bool,
+  className: PropTypes.string,
+  children: PropTypes.node,
+  help: PropTypes.node,
+  onChange: PropTypes.func,
+};
+
+RecommendedRepositorySetsToggler.defaultProps = {
+  enabled: false,
+  className: '',
+  children: __('Recommended Repositories'),
+  help: __('Some help to explain what am I'),
```

Comment:
We should probably update this with some actual help text :smile: @rohoover, thoughts on what this should say?

Reply:
I don't want to block this on waiting for text so maybe go with something like "This shows repositories that are used in a typical setup."  

Now that I think about it, should this be named "Popular Repositories"?  We can always change this when we hear back from @rohoover.